cmake/SetupSpec.cmake

```diff
@@ -4,6 +4,12 @@
 find_package(SpEC)
 
 if (NOT SpEC_FOUND)
+  # Pass `SPEC_PYTHONPATH` to cmake to make SpEC scripts available in Python
+  # independently of the SpEC exporter. These can be used until we have ported
+  # them to SpECTRE.
+  if (SPEC_PYTHONPATH)
+    set(PYTHONPATH "${SPEC_PYTHONPATH}:${PYTHONPATH}")
```

Comment:
I suggest you just move up the line that sets the pythonpath so it's always set, like this:

```cmake
find_package(SpEC)

# Make SpEC scripts available in Python. These can be used until we have ported
# them to SpECTRE.
if (SPEC_ROOT)
  set(PYTHONPATH "${SPEC_ROOT}/Support/Python:${PYTHONPATH}")
endif()

# That's it. No need to do anything else in this file.
```

Reply:
Oh, right. That's simpler. Done.